centreon/src/Core/Security/User/Application/UseCase/RenewPassword/RenewPassword.php

```diff
@@ -76,15 +80,26 @@ public function __invoke(
             return;
         }
 
+        try {
+            /** @var Configuration $providerConfiguration */
+            $providerConfiguration = $this->readConfigurationRepository->getConfigurationByType(Provider::LOCAL);
+            $this->info('Validate password against security policy');
+            $newPassword = UserPasswordFactory::create(
+                $renewPasswordRequest->newPassword,
+                $user,
+                $providerConfiguration->getCustomConfiguration()->getSecurityPolicy()
+            );
+        } catch(UserPasswordException | ConfigurationException $ex) {
+            $this->error("Unable to update password", ["trace" => (string) $ex]);
```

Comment:
Can you check the coding style with php-cs-fixer.
I guess this should use single quotes.
Example 👇🏻 
```suggestion
            $this->error('Unable to update password', ['trace' => (string) $ex]);
```